doc/source/contributor/environments/ci-multinode.rst

```diff
@@ -103,35 +110,36 @@ Then create a share type and share:
 
 .. code-block:: bash
 
-      manila type-create cephfs-type false --is_public true
-      manila type-key cephfs-type set vendor_name=Ceph storage_protocol=CEPHFS
-      manila create --name test-share --share-type cephfs-type CephFS 2
+      openstack share type create cephfs-type false --public true
+      openstack share type set cephfs-type --extra-specs vendor_name=Ceph
+      openstack share type set cephfs-type --extra-specs storage_protocol=CEPHFS
```

Comment:
nit, could've merged these into one line

Reply:
I tested and it doesn't work with either ``openstack share type set cephfs-type --extra-specs vendor_name=Ceph,storage_protocol=CEPHFS`` or `` openstack share type set cephfs-type --extra-specs vendor_name=Ceph --extra-specs storage_protocol=CEPHFS``

Reply:
[extra-specs](https://docs.openstack.org/python-manilaclient/latest/cli/osc/v2/index.html#share-type-set) is a list of key,value pairs: 
```
share type set
    [--extra-specs [<key=value> ...]]
    [--public <public>]
    [--description <description>]
    [--name <name>]
    <share_type>
```
So I reckon you'd be looking for: `openstack share type set cephfs-type --extra-specs vendor_name=Ceph,storage_protocol=CEPHFS`

Reply:
Note the space after the comma :)